runtime-testsuite/test/org/antlr/v4/test/runtime/go/BaseTest.java

```diff
@@ -464,13 +516,17 @@ private String locateGo() {
 		return prop;
 	}
 
-	private String locateRuntime() {
+	private static File locateRuntime() {
 		final ClassLoader loader = Thread.currentThread().getContextClassLoader();
 		final URL runtimeSrc = loader.getResource("Go");
```

Comment:
Never seen this before, so just curious what's going on here. A few more questions:

- How do you know it's defined then?
- Could it be a string from a config file?
- Does it define the path to the go installation?